app/fullService/api/assignAddressForAccount.ts

```diff
@@ -26,9 +27,18 @@ const assignAddressForAccount = async ({
   if (error) {
     // TODO - I'll write up a better error handler
     const errorMessage = error === 'Database(PasswordFailed)' ? 'Incorrect Password' : error;
+    logError(
+      errorMessage,
+      'app/fullService/api/assignAddressForAccount.ts:assignAddressForAccount'
+    );
     throw new Error(errorMessage);
   } else if (!result) {
-    throw new Error('Failure to retrieve data.');
+    const errorMessage = 'Failure to retrieve data.';
+    logError(
+      errorMessage,
+      'app/fullService/api/assignAddressForAccount.ts:assignAddressForAccount'
+    );
+    throw new Error(errorMessage);
```

Comment:
What would you think about including `throw new Error(errorMessage);` inside of the `logError` function? It looks like whenever there's an error we log and throw, so we could combine those into a single function